media/transcoder/src/test/java/com/example/transcoder/CreateJobFromAdHocTest.java

```diff
@@ -120,12 +120,23 @@ public void test_CreateJobFromAdHoc() throws Exception {
     assertThat(output, containsString(jobName));
     String[] arr = output.split("/");
     JOB_ID = arr[arr.length - 1].replace("\n", "");
-    bout.reset();
 
-    Thread.sleep(300000);
+    int attempts = 5;
+    boolean succeeded = false;
+
+    while (attempts > 0 && !succeeded) {
```

Comment:
Can we use a for loop here, so that `attempts` has smaller scope?  You can also break for `succeeded` as well, to eliminate the need for a variable